--- mgba-0.10.3/src/feature/updater-main.c.orig	2024-07-14 15:47:14.625249836 +0900
+++ mgba-0.10.3/src/feature/updater-main.c	2024-07-14 15:52:10.147277502 +0900
@@ -324,7 +324,7 @@
 		const char* argv[] = { qbin, NULL };
 		_execv(bin, argv);
 #elif defined(_POSIX_C_SOURCE) || defined(__APPLE__)
-		const char* argv[] = { bin, NULL };
+		char * const argv[] = { bin, NULL };
 		execv(bin, argv);
 #endif
 	}
