diff -ur code.old/AMF/AMFImporter.hpp code/AMF/AMFImporter.hpp
--- code.old/AMF/AMFImporter.hpp	2025-06-14 16:26:17.505105194 +0900
+++ code/AMF/AMFImporter.hpp	2025-06-14 16:28:18.273257250 +0900
@@ -62,6 +62,7 @@
 
 // Header files, stdlib.
 #include <set>
+#include <cstdint>
 
 namespace Assimp {
 
diff -ur code.old/AMF/AMFImporter_Node.hpp code/AMF/AMFImporter_Node.hpp
--- code.old/AMF/AMFImporter_Node.hpp	2025-06-14 16:26:17.505105194 +0900
+++ code/AMF/AMFImporter_Node.hpp	2025-06-14 16:28:38.474494981 +0900
@@ -54,6 +54,7 @@
 #include <list>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 // Header files, Assimp.
 #include "assimp/types.h"
diff -ur code.old/X3D/FIReader.hpp code/X3D/FIReader.hpp
--- code.old/X3D/FIReader.hpp	2025-06-14 16:26:17.521105310 +0900
+++ code/X3D/FIReader.hpp	2025-06-14 16:28:04.182098023 +0900
@@ -56,7 +56,7 @@
 #include <cwchar>
 #include <vector>
 //#include <stdio.h>
-//#include <cstdint>
+#include <cstdint>
 #ifdef ASSIMP_USE_HUNTER
 #  include <irrXML/irrXML.h>
 #else
